zipkin/src/test/java/zipkin/storage/DependenciesTest.java

```diff
@@ -95,12 +95,27 @@ public void getDependencies() {
    */
   @Test
   public void getDependencies_strictTraceId() {
-    List<Span> mixedTrace = new ArrayList<>(TRACE);
-    mixedTrace.set(1, TRACE.get(1).toBuilder().traceIdHigh(2).build());
+    List<Span> mixedTrace = asList(
+      Span.builder().traceIdHigh(1L).traceId(1L).id(1L).name("get")
```

Comment:
OPT: By extracting the traceId in a constant, it would emphasize that the trace is supposed to be considered the same